pytorch_lightning/loops/fit_loop.py

```diff
@@ -167,8 +168,18 @@ def run(self) -> None:
         if not self._should_skip_training():
             return super().run()
 
+    def create_progress(self):
+        self.training_loop.create_progress()
+
+        if getattr(self, "progress", None) is None:
```

Comment:
Define it in the __init__ so you don't need `getattr`

Reply:
We can't as we need the `optimizers` and `schedulers` to be defined. This should be cleaned out when

Reply:
Doen !